### ISSUE TITLE
.NET 4.7.2 support

### ISSUE THREAD

**Issue by gtimread** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#687]
Is .NET Framework 4.7.2 supported? The doc currently states ".NET 4.7.1 and higher (for Skype for Business Server CU 5 or later releases)"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7c475b5d-09ed-7f58-86d4-27f28e33ef7a
* Version Independent ID: f0ad50f3-ea93-9424-6de8-a0b778e90b23
* Content: [Server requirements for Skype for Business Server 2015](https://docs.microsoft.com/en-us/skypeforbusiness/plan-your-deployment/requirements-for-your-environment/server-requirements)
* Content Source: [Skype/SfbServer/plan-your-deployment/requirements-for-your-environment/server-requirements.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Skype/SfbServer/plan-your-deployment/requirements-for-your-environment/server-requirements.md)
* Product: **skype-for-business-itpro**
* GitHub Login: @MicrosoftHeidi
* Microsoft Alias: **heidip**

**Comment by gtimread** [comment]
Reason I ask for explicit guidance is because .NET 4.7.2 is still not supported by Exchange 2016: https://technet.microsoft.com/en-us/library/ff728623(v=exchg.150).aspx#Anchor_9

**Comment by AndreaBarr** [comment]
@gtimread Thank you for this feedback

I am gettting this .Net question over to our Teams/Skype team so that we can get this answered and the document updated so it reflects the anwer. 

I was able to get this into the backlog for the Teams team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
